surrealdb/clients/ws.py

```diff
@@ -82,14 +100,13 @@ async def connect(self) -> None:
 
         if self._token is not None:
             await self.authenticate(self._token)
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read

Reply:
In my opinion, it looks better. Lets have a discussion in the discord